string[] sensors
---
string sensors_values